src/nvim/ops.c

```diff
@@ -1169,8 +1162,7 @@ static void stuffescaped(char_u *arg, int literally)
  * If "regname" is a special register, return TRUE and store a pointer to its
  * value in "argp".
  */
-int 
-get_spec_reg (
+int get_spec_reg(
```

Comment:
sorry, I thought linter would only affected the closest 2-3 lines or so to a change and not a completely unchanged part of the file, but maybe that changed in the meanwhile (I was too busy the last 1-2 months to really follow all nvim discussions)